### PR TITLE
[14.0][IMP]cetmix_tower_server: Set server status from command

### PR DESCRIPTION
**Pull request by DemchukM**
Set server status from command

### CODE REVIEW

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Please add/update test coverage before submitting the code for review

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

**Comment by CetmixGitDrone** [comment]
Congratulations, PR rebased to [14.0-dev](https://github.com/cetmix/cetmix-tower/commits/14.0-dev).

File: cetmix_tower_server/models/cx_tower_server.py [hunk 1]
```diff
@@ -808,27 +808,41 @@ def _command_runner(
...
         elif command.action == "python_code":
-            return self._command_runner_python_code(
+            response = self._command_runner_python_code(
                 log_record,
                 rendered_command_code,
                 **kwargs,
             )
+            need_check_server_status = True
+
+        if need_check_server_status:
```
Comment (GabbasovDinar) [inline cetmix_tower_server/models/cx_tower_server.py]: Please, don't copy need_check_server_status variable, use `else` operator to set default value
 
```python
        response = None
        need_check_server_status = True

        if command.action == "ssh_command":
            response = self._command_runner_ssh(
                log_record,
                rendered_command_code,
                rendered_command_path,
                ssh_connection,
                **kwargs,
            )
        elif command.action == "file_using_template":
            response = self._command_runner_file_using_template(
                log_record,
                command.file_template_id,
                rendered_command_path,
                **kwargs,
            )
        elif command.action == "python_code":
            response = self._command_runner_python_code(
                log_record,
                rendered_command_code,
                **kwargs,
            )
        else:
            need_check_server_status = False

```

File: cetmix_tower_server/models/cx_tower_server.py [hunk 2]
```diff
@@ -808,27 +808,41 @@ def _command_runner(
...
+            if command.server_status and (
```
Comment (GabbasovDinar) [inline cetmix_tower_server/models/cx_tower_server.py]: please, use a single `if` clause:
```python
if need_check_server_status and command.server_status and ........
```

**Review by GabbasovDinar — CHANGES_REQUESTED** [review]

**Review by GabbasovDinar — APPROVED** [review]

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Review by ivs-cetmix — APPROVED** [review]
LGTM

**Comment by CetmixGitDrone** [comment]
This PR has the `approved` label and has been created more than 5 days ago. It should therefore be ready to merge by a maintainer (or a PSC member if the concerned addon has no declared maintainer). 🤖

**Comment by CetmixGitDrone** [comment]
What a great day to merge this nice PR. Let's do it!
Prepared branch [14.0-dev-ocabot-merge-pr-90-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-90-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 487e83dc7e3061342fc3c08634f8607f600ee6cd. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**